### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by georgebisbas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,17 +32,49 @@ lit tests/filecheck
...
-functionality, make sure to have clang, mlir-opt and mlir-translate installed
-and in the PATH. Clang can be installed by standard package managers (apt,
-pacman...), for MLIR follow [this](https://mlir.llvm.org/getting_started/) and
-run `ninja install` afterward. Given an input file `input.xdsl`, that contains
-IR with only the mirrored dialects found in `src/xdsl/dialects` (arith, builtin,
-cf, func, irdl, llvm, memref, and scf), run:
+xDSL can generate executables using MLIR as its backend.
+To benefit from this functionality, we first need to clone and build MLIR: https://mlir.llvm.org/getting_started/
+
+```bash
```
Comment (webmiche) [inline README.md:38]: Do we want this in our README? Isn't it enough to point to the `Getting Started` website of MLIR?

File: README.md [hunk 2]
```diff
@@ -32,17 +32,24 @@ lit tests/filecheck
...
-and in the PATH. Clang can be installed by standard package managers (apt,
-pacman...), for MLIR follow [this](https://mlir.llvm.org/getting_started/) and
-run `ninja install` afterward. Given an input file `input.xdsl`, that contains
-IR with only the mirrored dialects found in `src/xdsl/dialects` (arith, builtin,
-cf, func, irdl, llvm, memref, and scf), run:
+xDSL can generate executables using MLIR as its backend.
+To benefit from this functionality, we first need to clone and build MLIR.
+Please follow: https://mlir.llvm.org/getting_started/
+
+next, we need to have `mlir-opt`, `mlir-translate` and `clang` in the path
```
Comment (webmiche) [inline README.md:39]:
```suggestion
Next, we need to have `mlir-opt`, `mlir-translate` and `clang` in the path:
```

**Review by math-fehr — APPROVED** [review]
Besides Michel comment, looks good to me!

File: README.md (hunk 2 above)
Comment (georgebisbas) [reply README.md:39]: Fixed

**Comment by georgebisbas** [comment]
Fixed. SHould be ready to merge.

**Pull request merged**